### DISCUSSION TITLE
Stereographic projection for (100) direction

### DISCUSSION THREAD

**Discussion by inandi2** [mtex-toolbox/mtex#1903] [Ask Anything]
Dear community,
I am a new user of MTEX. I want to plot the stereographic projection for the (100) pole. The default projection I can plot is for (001) pole. 

![image](https://github.com/mtex-toolbox/mtex/assets/61553917/d04d16ac-9e54-43c5-b63b-911961274766)

I used the following lines:

`cs = crystalSymmetry('m-3m')`
`plot(cs,'complete','lineColor','k', 'linestyle',':','linewidth',1)`
`hold on;`
`h= Miller({1,0,0},{1,1,0},{1,1,1},{2,1,1},{1,2,2},cs)`
`for i = 1:length(h)`
` plot(h(i),'symmetrised','labeled','backgroundColor','w','doNotDraw','grid','upper')`
 ` hold all`
`end`

Also, I want to know how I can show the circle connecting pole (101) and (110).  Eventually, I want to plot something like below:

![image](https://github.com/mtex-toolbox/mtex/assets/61553917/8290a105-66f6-47e4-8233-b507fb0489c9)

Thank you for your help.

**Comment by kilir** [comment]
Hi,
rotating this plot is not implemented yet. In your particular case, you can work around it by simply changing the labels you assing, using  `'label', somestring` where somestring  is the appropriate label. But that is tedious. 

In order to draw lines between two points have a look at `Miller/line`.

Cheers,
Rüdiger
